mne/viz/_3d.py

```diff
@@ -1552,6 +1568,20 @@ def _plot_mpl_stc(stc, subject=None, surface='inflated', hemi='lh',
         time_viewer.subplots_adjust(left=0.12, bottom=0.05, right=0.75,
                                     top=0.95)
     fig.subplots_adjust(left=0., bottom=0., right=1., top=1.)
+
+    # add colorbar
+    from mpl_toolkits.axes_grid1.inset_locator import inset_axes
+    sm = plt.cm.ScalarMappable(cmap=cmap,
+                               norm=plt.Normalize(ctrl_pts[0], ctrl_pts[2]))
+    cax = inset_axes(ax, width="80%", height="5%", loc=8, borderpad=3.)
+    plt.setp(plt.getp(cax, 'xticklabels'), color='w')
+    sm.set_array(np.linspace(0, 1))
+    if colorbar:
+        cb = plt.colorbar(sm, cax=cax, orientation='horizontal')
+        cb_yticks = plt.getp(cax, 'yticklabels')
+        plt.setp(cb_yticks, color='w')
+        cax.tick_params(labelsize=16)
+        cb.patch.set_facecolor((0., 0., 0., 1.0))
```

Comment:
Use middle gray (`'0.5'`) not black

Reply:
It seems Pysurfer uses black. Don't we want to replicate that?

Reply:
It doesn't on master, it uses mid gray (or at least is coded to do so)